apps/sms/views/inbox/js/inbox.js

```diff
@@ -39,6 +39,11 @@ var InboxView = {
   // Set to |true| when in edit mode
   inEditMode: false,
 
+  /**
+   * Indicates whether user should be notified about newly saved draft.
+   */
+  notifyAboutSavedDraft: false,
```

Comment:
Soooo here we're - I found this approach as less intrusive as I can think of and as much similar to what we have now as possible. The only difference I can think of is that we'll show draft saved banner for one more new case:
* User enters composer;
* Edit draft;
* Taps on Home (draft is automatically saved, Drafts.on('saved') is handled in Inbox).
* User goes back to Inbox.

Previously user didn't see draft saved banner, now user will see it.

I'd say it's not a big deal, but tell me what you think? :)